inst/include/mode/mode.hpp

```diff
@@ -32,20 +32,24 @@ class container {
     return n_particles_;
   }
 
-  size_t n_state_full() {
-    return solver_[0].size();
+  size_t n_output_full() {
+    return m_.size() + m_.n_output();
```

Comment:
Have renamed these to distinguish between total output including derived values, and just the model size. Not sure about these names though, thoughts?